v2/api/containerservice/v1api20230315preview/fleets_update_run_spec_arm_types_gen.go

```diff
@@ -65,7 +65,9 @@ type ManagedClusterUpgradeSpec_ARM struct {
 	// KubernetesVersion: The Kubernetes version to upgrade the member clusters to.
 	KubernetesVersion *string `json:"kubernetesVersion,omitempty"`
 
-	// Type: ManagedClusterUpgradeType is the type of upgrade to be applied.
+	// Type: The upgrade type.
```

Comment:
Did this documentation change in the updated specs repo or do we have some race in our generator?

Reply:
That was the old submodule being lingering on. Thanks for pointing out, fixed it.